app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt

```diff
@@ -68,11 +69,19 @@ private fun setupMegazord() {
     // happens during a local substitution of application-services.
     // As a workaround, use reflections to conditionally initialize the megazord in case it's present.
     // See https://github.com/mozilla-mobile/reference-browser/pull/356.
-    try {
+    val is_megazord = try {
         val megazordClass = Class.forName("mozilla.appservices.ReferenceBrowserMegazord")
         val megazordInitMethod = megazordClass.getDeclaredMethod("init")
         megazordInitMethod.invoke(megazordClass)
+        true
     } catch (e: ClassNotFoundException) {
         Logger.info("mozilla.appservices.ReferenceBrowserMegazord not found; skipping megazord init.")
+        false
+    }
+    if (is_megazord) {
+        // We want rust logging to go through the log sinks.
+        // This has to happen after initializing the megazord, and
+        // it's only worth doing in the case that we are a megazord.
+        RustLog.enable()
```

Comment:
There's a `setupLogging` function that could do this (and get a boolean flag for `is_megazord`), but I don't care if you don't care.